reference/3.0/Microsoft.PowerShell.Core/Invoke-Command.md

```diff
@@ -1151,49 +1173,61 @@ returns the output of the invoked command, which is the value of the **ScriptBlo
-  results. Errors that would be terminating errors in a local command are treated as non-terminating
-  errors in a remote command. This strategy makes sure that terminating errors on one computer do
-  not close the command on all computers on which it is run. This practice is used even when a
-  remote command is run on a single computer.
-- If the remote computer is not in a domain that the local computer trusts, the computer might not
-  be able to authenticate the credentials of the user. To add the remote computer to the list of
-  trusted hosts in WS-Management, use the following command in the WSMAN provider, where
-  `<Remote-Computer-Name>` is the name of the remote computer:
-
-  `Set-Item -Path WSMan:\Localhost\Client\TrustedHosts -Value \<Remote-Computer-Name\>`
-- In Windows PowerShell 2.0, you cannot use the `Select-Object` cmdlet to select the
-  **PSComputerName** property of the object that `Invoke-Command` returns. Instead, to display the
-  value of the **PSComputerName** property, use the dot method to get the **PSComputerName**
-  property value ($result.PSComputerName), use a **Format** cmdlet, such as the `Format-Table`
-  cmdlet, to display the value of the **PSComputerName** property, or use a `Select-Object` command
-  where the value of the property parameter is a calculated property that has a label other than
-  PSComputerName. This limitation does not apply to Windows PowerShell 3.0 or later versions of
-  Windows PowerShell or PowerShell Core.
-- When you disconnect a **PSSession**, such as by using **InDisconnectedSession**, the session state
-  is Disconnected and the availability is None. The value of the **State** property is relative to
-  the current session. Therefore, a value of Disconnected means that the **PSSession** is not
-  connected to the current session. However, it does not mean that the **PSSession** is disconnected
-  from all sessions. It might be connected to a different session. To determine whether you can
-  connect or reconnect to the session, use the **Availability** property.
-
-  An **Availability** value of None indicates that you can connect to the session. A value of Busy
-  indicates that you cannot connect to the PSSession because it is connected to another session.
-
-  For more information about the values of the **State** property of sessions, see
-  [RunspaceState Enumeration](https://msdn.microsoft.com/library/system.management.automation.runspaces.runspacestate).
-
-  For more information about the values of the **Availability** property of sessions, see
-  [RunspaceAvailability Enumeration](https://msdn.microsoft.com/library/system.management.automation.runspaces.runspaceavailability).
+On Windows Vista, and later versions of the Windows operating system, to use the **ComputerName**
+parameter of `Invoke-Command` to run a command on the local computer, you must open PowerShell by
+using the Run as administrator option.
```

Comment:
Change to:

you must run PowerShell using the **Run as administrator** option.

Reply:
Fixed